lib/ts/recipe/session/constants.ts

```diff
@@ -14,4 +14,6 @@
  */
 
 export const REFRESH_API_PATH = "/session/refresh";
-export const HANDSHAKE_INFO_FILE_PATH = "/tmp/supertokens-handshakeInfo.json";
+const SERVERLESS_CACHE_BASE_FILE_PATH =
```

Comment:
Why is this being repeated here? Import from ../../constants?

Reply:
okay